vulnerablecode/settings.py

```diff
@@ -160,5 +160,5 @@
         "rest_framework.filters.SearchFilter",
     ),
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
-    "PAGE_SIZE": 100,
+    "PAGE_SIZE": 10,
```

Comment:
A comment about why we want to keep the default page size low would be useful.

Reply:
Added the comment